src/script/api/script_event_types.cpp

```diff
@@ -168,14 +168,13 @@ char *ScriptEventAdminPort::ReadString(HSQUIRRELVM vm, char *p)
 		p++;
 	}
 
-	*p = '\0';
-	sq_pushstring(vm, value, -1);
-	*p++ = '"';
+	sq_pushstring(vm, value, p - value);
```

Comment:
Maybe leave a comment for future-us telling that `sq_pushstring` does the right thing? Not sure that is really needed, so really a question :)

Reply:
I'm not a fan of writing comments to tell a function does the right thing and it "works". Though, with an extra local variable I can make it slightly more clear to the casual reader.